corehq/apps/sms/tasks.py

```diff
@@ -514,15 +513,6 @@ def _sync_user_phone_numbers(couch_user_id):
                     pass
 
 
-@no_result_task(queue='background_queue', acks_late=True,
```

Comment:
we'll need to keep the task around for one more deploy cycle because there might be tasks in the queue that need to be processed